File: LOGICAL/SWIFT/large-no-if-else.swift
import Foundation

print("Enter the First Number: ", terminator: "")
let num1 = Int(readLine()!)!

print("Enter the Second Number: ", terminator: "")
let num2 = Int(readLine()!)!

if num1 > num2 {
    print("\(num1) is Largest")
} else {
    print("\(num2) is Largest")
}

File: LOGICAL/JAVA/large-no-if-else.java
import java.util.Scanner;

class Scratch {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.println("Enter the First Number:");
        int num1 = scanner.nextInt();
        System.out.println("Enter the Second Number:");
        int num2 = scanner.nextInt();

        if (num1 > num2){
            System.out.println("Entered Number " + num1 + "is Largest");
        }else{
            System.out.println("Entered Number " + num2 + "is Largest");
        }

        scanner.close();
    }
}
